pkgs/development/python-modules/hsh/default.nix

```diff
@@ -0,0 +1,38 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, commandlines
+, unittestCheckHook
+, pexpect
+, naked
+, nix-update-script
+}:
+
+buildPythonPackage rec {
```

Comment:
```suggestion
buildPythonApplication rec {
```

Reply:
why? the binary is already properly wrapped

Reply:
Everything under pythonPackages must be a library to be able to be consumed by other packages

Reply:
backgroundremover is both.  The primitives are exposed to use in other stuff

Reply:
Both a library and binary

Reply:
I have been asking you for some time to explain that backgroundremover can also be used as a library.
Not subjectively, but objectively.

Reply:
I added a example script as a `passthru.test`. Also hardcoded the default model paths so it works properly in a derivation.

It has some rough edges but works nice.

Reply:
Thanks for writing the test. Great work.
However, after reading this code, I'm strongly convinced that this is not a library, at least at this point.
Python does not have private objects as a language specification, so you can access all functions and methods, even if they are not intended to be exposed.
In this example, you are just accessing an unstable internal function that has no API documentation, no signature, and no simple docstring.
If you really want to use it as a library in your own scripts, please use `toPythonModule`.

Reply:
Done in binary way